mis_builder_analytic_filter/static/src/js/mis_builder.js

```diff
@@ -0,0 +1,78 @@
+openerp.mis_builder_analytic_filter = function(instance) {
```

Comment:
shouldnot we need copyright in js files, following https://github.com/OCA/maintainer-tools/blob/master/template/module/static/src/js/module_name.js